lib/Fhp/BaseAction.php

```diff
@@ -123,6 +124,9 @@ public function needsTan(): bool
         return !$this->isAvailable && $this->tanRequest !== null;
     }
 
+    /**
+     * @return TanRequest|HITANv6|null
```

Comment:
Why is `HITANv6` on this list?

Reply:
Probably you want to add `TanRequest::getChallengeHhdUc()` or even `TanRequest::getChallengeAsImage()` and do the conversion to the image class in there.

Reply:
The interface was not changed, because the old class "Response\GetTanRequest" has this class, but it was not migrated to FinTSNew. I will add this.